File: serve_icons.js
const http = require("http");
const fs = require("fs");
const path = require("path");
const root = path.join(__dirname, "public");
const mime = { ".html": "text/html", ".png": "image/png", ".json": "application/json" };
http.createServer((req, res) => {
  const file = path.join(root, req.url === "/" ? "/index.html" : req.url);
  fs.readFile(file, (err, data) => {
    if (err) { res.writeHead(404); res.end("Not found"); return; }
    res.writeHead(200, { "Content-Type": mime[path.extname(file)] || "text/plain" });
    res.end(data);
  });
}).listen(7723, () => console.log("ready"));

File: serve_icons.py
import os, http.server, socketserver
os.chdir(os.path.join(os.path.dirname(__file__), "public"))
socketserver.TCPServer.allow_reuse_address = True
with socketserver.TCPServer(("", 7723), http.server.SimpleHTTPRequestHandler) as h:
    h.serve_forever()
